src/test/java/uk/ac/ebi/ampt2d/accession/generator/TestPrefixAccessionGenerator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.generator;
+
+import uk.ac.ebi.ampt2d.accession.AccessionGenerator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class TestPrefixAccessionGenerator implements AccessionGenerator<String> {
```

Comment:
This kind of dummy classes could be placed in [a `test` package](https://github.com/EBIvariation/eva-pipeline/tree/develop/src/test/java/uk/ac/ebi/eva/test) to avoid mixing them with the test classes. A package hierarchy could still be replicated under `test` if that helps.